product_docs/docs/pgd/5/quick_start_aws.mdx

```diff
@@ -131,60 +131,61 @@ tpaexec configure democluster \
   --no-git \
   --hostnames-unsorted
 ```
-We included options to specify using AWS, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD Proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
+These options specify to use AWS, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
```

Comment:
These options specify that TPA should create a configuration that uses AWS, in a single location, with three data nodes...

Reply:
Please don't use "should." Revise to These options specify for TAP to create ...